Source/inv.cpp

```diff
@@ -887,7 +887,7 @@ void CheckInvCut(int pnum, Point cursorPosition, bool automaticMove, bool dropIt
 		}
 	}
 
-	if (dropItem) {
+	if (dropItem && !holdItem.isEmpty()) {

```

Comment:
Once again I wonder: wouldn't it make more sense for the empty check to be inside the `TryDropItem` function instead?

Reply:
@yuripourre can you apply the suggestion? :P